Gruntfile.js

```diff
@@ -57,9 +57,9 @@ module.exports = function (grunt) {
         grunt.log.subhead('Running Grunt in DEV mode');
     }
 
-    // Default task
+    // Default task - used by grunt-tc
```

Comment:
yeah potentially. i wanted to limit this to (hopefully) non-breaking stuff. assume `make` is available there etc, but was't sure. happy to change if anyone (@rich-nguyen?) can confirm it should be ok?